applications_tests/gls_navier_stokes_3d/poiseuille_restart_files/poiseuille_restart.flowcontrol

```diff
@@ -1,5 +1,5 @@
 Flow control
 Previous_beta 25.9666
-Previous_flow_rate -10.0126
+Previous_average_velocity -3.20768
```

Comment:
The reason for the change is the switch from flow to average velocity right?

Reply:
Yes, it's -10.0126/3.12145 (flow rate/area)